modules/ROOT/pages/policy-mule3-add-remove-headers.adoc

```diff
@@ -111,23 +115,8 @@ The policy you applied moves from Available Policies to the Applied Policies lis
 
 === Applying the Remove Request/Response Header Policies
```

Comment:
It should be "Header Removal policy" (here and in the text)

Reply:
Ok! Done!